### ISSUE TITLE
Unable to Invoke lambda function

### ISSUE THREAD

**Issue by praisegeek** [ConduitVC/aws-utils#162]
LambdaRunner logs the error below when invoking Lambda functions from resolver templates.

```code
Assertion failed: (!uv__is_closing(handle)), function uv_close, file ../deps/uv/src/unix/core.c, line 117.
```

Using serverless-webpack-plugin to compile files.

Environment:

Mac OSX
serverless: 1.49.0
appsync-emulator-serverless: 0.14.2 ( tested on 0.14.0 as well )

**Comment by praisegeek** [comment]
Follow up  #152

**Comment by praisegeek** [comment]
Fixed in #163
